generators/client/templates/react/src/main/webapp/app/shared/auth/private-route.spec.tsx.ejs

```diff
@@ -14,24 +17,47 @@ const TestComp = () => <div>Test</div>;
 describe('private-route component', () => {
 <%_ if (enableTranslation) { _%>
   beforeAll(() => {
-    TranslatorContext.registerTranslations('<%= nativeLanguage %>', {});
+    TranslatorContext.registerTranslations('<%= nativeLanguage %>', {
+      'error.http.403': 'You are not authorized to access this page.',
+    });
   });
-
 <%_ } _%>
+
+  const mockStore = configureStore([thunk]);
+  const wrapper = (Elem: JSX.Element, authentication) => {
+    const store = mockStore({ authentication });
+    return render(<Provider store={store}>{Elem}</Provider>);
+  };
```

Comment:
Honestly, I'd just use your full "real" redux store with `preloadedData` instead of a mock store you have to set up with manual logic for each test

Reply:
Yes, but I would have to change the state for each test anyway so not much different in the end IMO